README.md

```diff
@@ -1,14 +1,5 @@
 # Salesforce APIs for Postman
 A collection of [Postman](https://www.postman.com/) resources for the Salesforce APIs:
-- Auth
```

Comment:
I recommend to keep the list of supported API here so that users can see at a glance all features. There's a low chance people scroll till the end of the readme for it.